.github/workflows/on_prerelease.yaml

```diff
@@ -13,4 +13,6 @@ jobs:
     with:
       tag: ${{ github.event.release.tag_name }}
       integration: "mssql"
+      upload_fips_packages: true
+      dest_prefix: "sairaj-fips-test-mssql"
```

Comment:
You should remove the dest_prefix, it's meant only for testing but should be removed now we want to merge it